crates/nostr/examples/nip05.rs

```diff
@@ -20,5 +20,8 @@ fn main() -> Result<()> {
         println!("NIP-05 NOT verified");
     }
 
+    let profile = nip05::get_profile_blocking("_@fiatjaf.com", None)?;
+    println!("Profile example (including relays): {profile:#?}");
+
```

Comment:
To leave this example, add `base` feature in the `nip05` example in the `Cargo.toml` file.